### DISCUSSION TITLE
Is this a reaktoro versions or a Python problem?

### DISCUSSION THREAD

**Discussion by mikewxyz** [reaktoro/reaktoro#388] [Q&A]
### Description

I installed reaktoro following the advice on the website (I'm using Linux).  I then started trying to run some of the tutorials from the website, but I can't get them to work.

This looks to me like a version problem - the turorial problems are for v1 of reaktoro but reaktoro has moved on?  Where can I find up to date tutorial examples or v1 of reaktoro so these old tutorial examples will work.

(I'm an ancient Fortran/ C programmer so I'm not exactly up to speed on OOP stuff in Python yet.)

### Example

from reaktoro import *
db  = reaktoro4py.Database('Supcrt')
print(f"{'Species':<20}{'Formula':<60}{'Molar Mass (kg/mol)':<20}")
for species in db.species():
    print(f"{species.name():<20}{species.formula().str():<60}{species.molarMass():<20.6f}")

and the first  error message I'm getting:  (after a few iterations of changing the db= statement to try and get it to work) is:

"TypeError: __init__(): incompatible constructor arguments.  The following argument types are supported:
    1. reaktoro.reaktoro4py.Database()
    2. reaktoro.reaktoro4py.Database(arg0: ...)
 etc.

### What is your operating system?

Linux

**Comment by allanleal** [comment]
Hi @mikewxyz - on the website, https://reaktoro.org, all tutorials are for Reaktoro v2. You can start with the basic tutorials here:
https://reaktoro.org/tutorials/basics/index.html

![image](https://github.com/reaktoro/reaktoro/assets/5825588/0c39ea67-9617-46e0-bce1-3f40a9607c79)

Hope this helps.

**Reply by mikewxyz** [reply]
Thanks, things are starting to work now!
